src/transformState.js

```diff
@@ -5,7 +5,27 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  const res = state;
```

Comment:
i'm pretty sure if you `console.log(res, state)` they will be same
that's because res is a reference to state, but not the standalone variable
you better spread the state:
```suggestion
  const res = [...state];//this will create an individual variable instead of a reference
```